packages/vite/src/node/plugins/importMetaGlob.ts

```diff
@@ -168,29 +171,38 @@ export async function parseImportGlob(
     if (ast.arguments.length < 1 || ast.arguments.length > 2)
       throw err(`Expected 1-2 arguments, but got ${ast.arguments.length}`)
 
-    const arg1 = ast.arguments[0] as ArrayExpression | Literal
+    const arg1 = ast.arguments[0] as ArrayExpression | Literal | TemplateLiteral
     const arg2 = ast.arguments[1] as Node | undefined
 
     const globs: string[] = []
-    if (arg1.type === 'ArrayExpression') {
-      for (const element of arg1.elements) {
-        if (!element) continue
-        if (element.type !== 'Literal') throw err('Could only use literals')
+
+    const validateLiteral = (element: Expression | SpreadElement) => {
+      if (!element) return
```

Comment:
```suggestion
```
I think we could remove this condition since `element` is never `null`.

Reply:
Should I delete line 201 instead of 180, after which I may no longer need to repeat the behavior of line 201, what do you think?

Reply:
That sounds better to me.